qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/offers/OfferToPDFConverter.groovy

```diff
@@ -151,7 +151,14 @@ class OfferToPDFConverter implements OfferExporter{
     void setQuotationDetails(){
         htmlContent.getElementById("offer-identifier").text(offer.identifier.toString())
         htmlContent.getElementById("offer-expiry-date").text(offer.expirationDate.toLocalDate().toString())
-        htmlContent.getElementById("offer-date").text(offer.modificationDate.toLocalDate().toString())
+        Date offerDate = offer.modificationDate
+        htmlContent.getElementById("offer-date").text(String.format(
+                "%s, %d %s %d",
+                offerDate.toDayOfWeek().name(),// write lowercase
+                offerDate.toDayOfWeek().value,
+                offerDate.toMonth().name(), //write lowercase
+                offerDate.toYear().value
+        ))
```

Comment:
Maybe consider using the DateFormat https://www.geeksforgeeks.org/dateformat-format-method-in-java-with-examples/

Reply:
did that, thanks for your suggestion. Looks cleaner now :)